client/src/components/Profile/EditPasswordForm/EditPasswordForm.stories.js

```diff
@@ -0,0 +1,10 @@
+import EditPasswordForm from './EditPasswordForm'
+
+export default {
+  title: 'Component/Profile/EditPasswordForm',
+  component: EditPasswordForm,
+}
+
+const Template = () => <EditPasswordForm />
+
+export const EditPasswordFormComplete = Template.bind({})
```

Comment:
I think you could add an action here. Something like
EditPasswordFormComplete.args = {
onSubmit: action(submit => comsole.log(submit))

Reply:
Changed it!